### ISSUE TITLE
Feature: Add Log Probabilities to the response

### ISSUE THREAD

**Issue by ardan-bkennedy** [ardanlabs/kronk#24]
This API call will provide the Logits for each bat
llama.GetLogits(ctx llama.Context, nTokens int, nVocab int)

GetLogits retrieves all token logits from the last call to llama_decode. Returns a slice of float32 of length nTokens * nVocab, or nil if not available.
